logpyle/__init__.py

```diff
@@ -33,8 +33,6 @@
 .. autofunction:: add_simulation_quantities
 """
 
-__copyright__ = "Copyright (C) 2009-2013 Andreas Kloeckner"
```

Comment:
There's no reason to remove this. Copyright statements serve to track file authorship and don't get modified, only added to.

Reply:
Ok, so should this get the same line as in the LICENSE file?

Reply:
It records distinct information from the LICENSE file. Technically, everyone contributing to each file should make sure their copyright is included at the top of the file. (Or, for CEESD, that the board of trustees copyright is included).

Reply:
Ok, removed this change for now.